seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -3991,7 +3991,7 @@
       "tests": [
         "collection = collectionCopy; assert(update(5439, \"artist\", \"ABBA\")[5439][\"artist\"] === \"ABBA\", 'message: After <code>update(5439, \"artist\", \"ABBA\")</code>, <code>artist</code> should be <code>\"ABBA\"</code>');",
```

Comment:
This test fail because <code>update(5439, \"artist\", \"ABBA\")[5439][\"artist\"]</code> is <undefined>.

You must run <code>update(5439, \"artist\", \"ABBA\");</code> and then <code>assert(collection[5439][\"artist\"] === \"ABBA\", 'message: After <code>update(5439, \"artist\", \"ABBA\")</code>